classes/module/Module.php

```diff
@@ -758,7 +758,11 @@ public function enable($force_all = false)
             }
         }
 
-        if ($this->getOverrides() != null) {
+        if (count($items) == Shop::getTotalShops()) {
+            $force_all = true;
```

Comment:
@LittleBigDev these are the wrong commits I think:
[https://github.com/PrestaShop/PrestaShop/commit/f1fdee320c6713b67ff37b2eb808bf761e9859fd](url)
[https://github.com/PrestaShop/PrestaShop/commit/7d6c52da604c677ae4e454329313904e166fbe5f](url)
As you said enabling/disabling a module should only affect its behavior and not its overrides unless it is enabled or disabled on all shops.